src/core/gltf2importer/meshparser_utils.cpp

```diff
@@ -376,8 +376,31 @@ bool vertexSizesForAttributesAreValid(const QVector<Qt3DRender::QAttribute *> &a
 namespace Kuesa {
 namespace GLTF2Import {
 namespace MeshParserUtils {
-Qt3DRender::QAttribute *createTangentAttribute(Qt3DRender::QGeometry *geometry, Qt3DRender::QGeometryRenderer::PrimitiveType primitiveType)
```

Comment:
I would leave createTangentAttribute like it was before and instead create a function hasTangentAttribute. Otherwise we have a function returning nullptr for various conditions (missing geometry, non triangle primitive type ...)

Reply:
right, makes sense!